shared/Microsoft.AspNetCore.HttpSys.Sources/RequestProcessing/NativeRequestContext.cs

```diff
@@ -377,18 +315,64 @@ private static SocketAddress CopyOutAddress(IntPtr address)
             return null;
         }
 
+
+        internal unsafe WindowsPrincipal GetUser()
```

Comment:
You can move these to their original position so we can see the real diff.
I don't think you need `unsafe`.